src/shared/rpc/methods/sign-stacks-transaction.ts

```diff
@@ -0,0 +1,40 @@
+import { DefineRpcMethod, RpcRequest, RpcResponse } from '@btckit/types';
+import { StacksNetworks } from '@stacks/network';
+import * as yup from 'yup';
+
+import { formatValidationErrors, getRpcParamErrors, validateRpcParams } from './validation.utils';
+
+const rpcSignStacksTransactionParamsSchema = yup.object().shape({
+  stxAddress: yup.string().required(),
+  serializedTransactionHex: yup.string().required(),
```

Comment:
I wonder if just `transactionHex` is okay? Are there other ways to format a stacks transaction in hex other than to serialise it? If not, it's not necessary to specify that it must be serialised?